app/magisk/META-INF/com/google/android/update-binary

```diff
@@ -0,0 +1,173 @@
+#!/sbin/sh
```

Comment:
Here's an update of Magisk update_binary.

https://github.com/topjohnwu/Magisk/blob/master/scripts/module_installer.sh